robot_skills/src/robot_skills/arms.py

```diff
@@ -341,22 +373,28 @@ def handover_to_robot(self, timeout=10):
             return False
 
     def handshake(self, timeout=10):
```

Comment:
Why do we even have this method? Could be replaced with ```arm.send_joint_trajectory('handshake', timeout)```, which is also a one-liner

Reply:
I did a cleanup, I didn't want to remove the function. but I agree that it could be removed